pepper-apis/src/main/java/org/broadinstitute/ddp/db/TransactionWrapper.java

```diff
@@ -372,6 +372,7 @@ private HikariDataSource createDataSource(int maxConnections, String dbUrl, DB d
         config.addDataSourceProperty("prepStmtCacheSize", "250");
         config.addDataSourceProperty("prepStmtCacheSqlLimit", "2048");
         config.addDataSourceProperty("cacheServerConfiguration", "true");
+        config.addDataSourceProperty("useCursorFetch", "true");
```

Comment:
Can't figure out what this option does when we don't use the setFetchSize option for queries. Do you know?

Reply:
Based on the documentation that I linked in the PR description, it uses cursor-based fetching when `setFetchSize() > 0`. So when we don't have `setFetchSize()` I'll assume it uses default behavior, which is fetching all the rows in resultset.